### PR TITLE
Created hollaATyacodegirl's first post!

### PR DESCRIPTION
**Pull request by hollaATyacodegirl**
I added a post about hollaATyacodegirl and her Python experience.

### CODE REVIEW

File: _posts/hollaATyacodegirl/2018-05-17-hollaATyacodegirl-first-post.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+author: hollaATyacodegirl
+title: "hollaATyacodegirl's first post!"
+---
+
+Hi! I'm codegirl of hollaATyacodegirl, AKA Halina. I was self-taught in Python up to a beginner-intermediate level, and so I am very excited
+to take a formal course in Python. Let's begin!
```
Comment (eah13) [inline _posts/hollaATyacodegirl/2018-05-17-hollaATyacodegirl-first-post.md]: 🚀 let's go!

Comment (hollaATyacodegirl) [reply _posts/hollaATyacodegirl/2018-05-17-hollaATyacodegirl-first-post.md]: 👍 👍 👍

File: _posts/hollaATyacodegirl/2018-05-17-hollaATyacodegirl-first-post.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+author: hollaATyacodegirl
+title: "hollaATyacodegirl's first post!"
```
Comment (hollaATyacodegirl) [inline _posts/hollaATyacodegirl/2018-05-17-hollaATyacodegirl-first-post.md]: glad these caps are accurate

**Comment by hollaATyacodegirl** [comment]
Here's a link to my Turtle Basics Trinket: https://trinket.io/python/e86c638466

**Pull request merged**